package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@angular-architects/module-federation": "14.3.14",
     "@angular/animations": "16.2.1",
-    "@angular/cdk": "16.2.1",
+    "@angular/cdk": "^17.0.3",
```

Comment:
We should 1) pin this version and 2) pick the same version as other Angular components ("16.2.1")